src/library/defeq_canonicalizer.cpp

```diff
@@ -11,37 +11,37 @@ Author: Leonardo de Moura
 #include "library/cache_helper.h"
 
 namespace lean {
-struct defeq_canonize_cache {
+struct defeq_canonicalize_cache {
```

Comment:
Canonizer is the name commonly used:

http://link.springer.com/chapter/10.1007%2F978-3-540-31862-0_27
http://arxiv.org/pdf/1207.3262.pdf
https://books.google.pt/books?id=B_tX0BdH9ScC&pg=PA442&lpg=PA442&dq=Shostak+canonizer&source=bl&ots=RnC-WVphDL&sig=4rN_JrRuw0ar5YlkE8jqkRpWQbU&hl=en&sa=X&ved=0ahUKEwjv-ePd89XNAhUqDMAKHQz3CMY4ChDoAQgsMAI#v=onepage&q=Shostak%20canonizer&f=false